config/application.rb

```diff
@@ -13,5 +13,9 @@
   
     # Compress the responses to reduce the size of html/json controller responses.
     config.middleware.use Rack::Deflater
+
+    # Configure the autoload and eager load paths
+    config.autoload_paths << Rails.root.join('lib')
+    config.eager_load_paths << Rails.root.join('lib')
```

Comment:
Eager loading the whole `lib` directory is usually bad practice. Instead it would be better to load `lib/<APP_DIRECTORY_NAME>` e.g. `lib/mia-web`